### DISCUSSION TITLE
Limit Access per user / group

### DISCUSSION THREAD

**Discussion by diegoweb** [blakeblackshear/frigate#15333] [Ideas]
Hey everyone!

Well, I'm running 1 frigate instance at home and via VPN I connect to other sites and record the cameras from there as well.
Basically, since Frigate has an user managment system, it would be awesome if we could set groups or per user access to allow watching live feed camera and features like playback, max days of history playback, and what cameras or group of camera it would have access.
This would allow to install off-site camera monitors and limit their acess to their camera only.

And in case of unauthorized access to that place, none of my home cameras or other places would be exposed.

And if possible, since this user would have limited access, allow it to log via an easier way (or via cookies with long expire time), so we could set up a simple raspberry to open a browser and have that placed already logged in and show whatever access they would have.

**Comment by hawkeye217** [comment]
Roles/groups already has a pinned feature request and is planned for a future version. This is a duplicate of https://github.com/blakeblackshear/frigate/issues/6614
